README.md

```diff
@@ -21,6 +21,10 @@ have permissions that you may not have.
       Glide 0.13.1 is known to be broken.
    1. [jq](https://stedolan.github.io/jq/) (can also use package manager)
 
+   Development helper scripts assume an up-to-date GNU tools environment. Recent Linux distros should work out-of-the-box.
```

Comment:
Copy from the [Kubernetes docs](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment)